content/applications/sales/crm/acquire_leads/generate_from_website.rst

```diff
@@ -1,78 +1,105 @@
+Odoo’s ready-to-use contact form to generate leads/opportunities.
 
-Create a custom contact form
-============================
+.. image:: generate_from_website/default-contact-us-page.png
+   :align: center
+   :alt: Default Contact Us page
 
-You may want to know more from your visitor when they use the contact form to
-you will need to build a custom contact form on your
-website. These contact forms can generate multiple types of records in
-the system (emails, leads/opportunities, project tasks, helpdesk
-tickets, etc.)
+This can be activated or deactivated at any time by going to :menuselection:`Website --> Go to
+Website --> Customize --> Contact Form` on your website and toggling the option on or off.
 
-Configuration
-=============
+.. image:: generate_from_website/contact-form-toggle.png
+   :align: center
+   :alt: Contact Form toggle
 
-You will need to install the free *Form Builder* module, only
-available in Odoo Enterprise.
+When the form is toggled off, the *Contact Us* page lets visitors know that they can contact your
+company by email contact and display a button to send one directly. Any emails sent this way will
+generates a lead/opportunity.
 
-.. image:: media/generate_from_website05.png
+.. image:: generate_from_website/default-contact-us-page-no-form.png
    :align: center
+   :alt: Contact Us Page using email
 
-Create a custom contact form
-----------------------------
-
-First, go to the page where you want to put your contact form. In edit mode, drag
-the form builder onto the page, and you will be able to add all the fields
-you wish.
+To change the form to a specific sales channel, go to :menuselection:`Website --> Configuration -->
+Settings.` Under *Communication,* you can change which *Sales Team* or *Salesperson* is assigned
+to any incoming leads or opportunities from your Contact Form.
 
-.. image:: media/generate_from_website06.png
+.. image:: generate_from_website/contact-form-settings.png
    :align: center
+   :alt: Contact Form settings
 
-By default, any new contact form will send an email. You can switch to
-lead/opportunity generation in *Change Form Parameters*.
+Creating a custom Contact Form
+==============================
 
-.. note::
-   If the same visitor uses the contact form twice, the second
-   entry will be added to the first lead/opportunity in the chatter.
+Wanting more info from visitors using the Contact Form? Contact Forms can be customized for the
+specific information your team needs. They can generate multiple types of records in the
+system, such as emails, leads/opportunities, project tasks, helpdesk tickets, and more.
+
+The free *Form Builder* module can be installed from either the apps page or automatically when
+adding a form to a page in the Website Builder. Adding a whole new form can be great if you are
+soliciting certain information that needs filtering, such as suggestions, bug reports, etc.
```

Comment:
Reminder, be mindful of _continuous_ present tense.
Would do this instead:
```
The *Form Builder* module is installed automatically when a Form element is added to a web page via
the Website Builder. The Form Builder can also be installed manually from the Apps page.
```

Reply:
@StraubCreative The edits to this paragraph didn't make it onto the latest commit either. Here's the final edited version:
```suggestion
The *Form Builder* module is installed automatically when a Form element is added to a web page via the Website Builder. It can also be installed manually from the :guilabel:`Apps` page.
```